components/image-slideshow.tsx

```diff
@@ -4,65 +4,40 @@ import { useEffect, useState } from "react";
 
 import { Card, CardContent, CardDescription, CardTitle } from "@/components/ui/card";
 import Link from "next/link";
-
+import React from "react";
+import { useNewsData } from "@/app/api/fetchNewsData/fetchNews";
 
 const ImageSlideshow = () => {
 
-  const [news, setNews] = useState({
-      title: "Pictures from space show mighty smog choking Lahore",
-      imageSrc: "https://ichef.bbci.co.uk/news/1024/branded_news/dfac/live/77f149a0-9dc3-11ef-935d-3107d1c873e8.jpg",
-      url: "https://www.bbc.com/news/articles/cm20k76d5xno"
-  });
-  const [error, setError] = useState<string | null>(null); 
-  const [loading, setLoading] = useState<boolean>(true); 
-  const apiKey = 'f448d8f760cb4fa8b7595fe1400d8389'
-
-  const fetchNews = async () => {
-    setError(null);
-    setLoading(true)
-    
-    const url = "/api/fetchNewsData"
+  const [title, setTitle] = useState<string>("Pictures from space show mighty smog choking Lahore")
```

Comment:
It is not necessary to mention type <string> here as default value's type is used which is already string here. Same thing for below.